### DISCUSSION TITLE
Another FreshRSS misconfig or feed problem?

### DISCUSSION THREAD

**Discussion by squromiv** [FreshRSS/FreshRSS#4556] [Q&A] [answered]
I am testing FreshRSS v. 1.20.0 dev.
[This feed](https://www.yaplakal.com/news.xml) can not be added in it. But other feed-readers in the same machine are able to do. As for example: [QuiteRSS](https://quiterss.org/), [Feedbro Firefox addon](https://addons.mozilla.org/de/firefox/addon/feedbroreader/). It may be Cloudflare specificity, but how to overcome this limitation?

**Comment by Alkarex** [comment, marked as answer]
Yes, it looks like various requests are blocked by Cloudflare (they keep breaking the Web in various ways).
Please try a cURL command from your machine (with various options). If that does not work, it will not work from FreshRSS either.

```sh
curl -v -A 'curl/7.74.0' 'https://www.yaplakal.com/news.xml'
```

The best is to contact the feed owner.

**Comment by Alkarex** [comment]
Related issue https://github.com/FreshRSS/FreshRSS/issues/4323
